src/tests/validity/mv.ab

```diff
@@ -0,0 +1,3 @@
+unsafe $touch a$
```

Comment:
```suggestion
unsafe $cd /tmp$
unsafe $touch a$
```

Reply:
Thank you for the suggestion! I've applied your changes as requested.